bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties

```diff
@@ -0,0 +1,6 @@
+channel-type.system.signal-strength.label = Signalst�rke
+channel-type.system.low-battery.label = Niedriger Batteriestatus
+channel-type.system.battery-level.label = Batterie Level
```

Comment:
Hmmm, that somehow sounds "Denglisch" to me. The internet comes up with "Batteriezustand" or "Batterieladezustand". Not sure if they are better...